packages/blocks/src/store/selectors.js

```diff
@@ -95,6 +95,44 @@ export function getBlockVariations( state, blockName, scope ) {
 	} );
 }
 
+/**
+ * Returns the active block variation for a given block based on its attributes.
+ *
+ * @param {Object}                state      Data state.
+ * @param {string}                blockName  Name of block (example: “core/columns”).
+ * @param {Object}                attributes Block attributes used to determine active variation.
+ * @param {WPBlockVariationScope} [scope]    Block variation scope name.
+ *
+ * @return {(WPBlockVariation|void)} Active block variation.
+ */
+// eslint-disable-next-line camelcase
+export function unstable__getActiveBlockVariation(
+	state,
+	blockName,
```

Comment:
One thing to consider is whether we could combine `blockName` and `attributes` into a single param. 

```js
export function unstable__getActiveBlockVariation( state, { name, attributes}, scope );
```

With that instead of doing:

```js
const { attributes, name: blockName } = block;
const match = getActiveBlockVariation( blockName, attributes );
```

we could do:
```js
const match = getActiveBlockVariation( block );
```

Reply:
Do we have any other instances where we do the same - block name + attributes under one object? So far I've only seen them separately like one param `block`/`blockName` and one param `attributes`.

Reply:
Let's leave it as is then 👍🏻